Framework/Algorithms/test/NRCalculateSlitResolutionTest.h

```diff
@@ -4,26 +4,19 @@
 #include <cxxtest/TestSuite.h>
 
 #include "MantidAlgorithms/NRCalculateSlitResolution.h"
-#include "MantidDataObjects/Workspace2D.h"
-#include "MantidAPI/AnalysisDataService.h"
-#include "MantidAPI/Run.h"
 #include "MantidGeometry/Instrument.h"
-#include "MantidGeometry/Instrument/Detector.h"
-#include "MantidKernel/PropertyWithValue.h"
 #include "MantidKernel/TimeSeriesProperty.h"
-#include "MantidKernel/V3D.h"
```

Comment:
With headers the ideal is to 'Use-What-You-Include'. As you use `V3D` in this file it should be left in the header, the same possibly for some others. Removing the truly redundant includes is a nice clean-up though :+1: . See - https://github.com/include-what-you-use/include-what-you-use#instructions-for-users and https://www.mantidproject.org/IWYU .

Reply:
I will try this tool, thanks for this information!